core/db/migrate/20151026093607_change_return_item_pre_tax_amount_to_amount.rb

```diff
@@ -0,0 +1,19 @@
+class ChangeReturnItemPreTaxAmountToAmount < ActiveRecord::Migration
+  def up
+    execute(<<-SQL)
+      UPDATE spree_return_items
+      SET pre_tax_amount = pre_tax_amount + included_tax_total;
+    SQL
```

Comment:
I added a safeguard clause that sets included_tax_total to 0 if `NULL`. The `pre_tax_amount`/ `amount` column has a `NOT NULL` db constraint set, so no worries on that side of the plus.